tests/test_bs.py

```diff
@@ -423,415 +420,96 @@ def _run_curl_pod():
 #    TESTS
 # ==============================================================================
 
-dt = datetime.now()
-today_date = dt.strftime("%Y.%m.%d")
-current_index = 'kubernetes_cluster-' + today_date
```

Comment:
It looks like making this variable non-global broke `hare`, `sync` and `late nodes` tests (see [results](https://travis-ci.org/spacemeshos/go-spacemesh/builds/639364172)). Either make it global again, or fix the other tests some other way.